doc/classes/CollisionPolygon.xml

```diff
@@ -20,7 +20,7 @@
 			If true, no collision will be produced.
 		</member>
 		<member name="polygon" type="PoolVector2Array" setter="set_polygon" getter="get_polygon">
-			Array of vertices which define the polygon.
+			Array of vertices which define the polygon.  Note that the returned value is a Copy-on-Write clone of the original.  Methods which mutate the size or properties of the return value will not impact the original polygon.  To change properties of the polygon, assign it to a temporary variable and make changes before calling set_polygon.
```

Comment:
Style issue: there should be only one space after dots. Same for other changed paragraphs.

Setters for properties are kind of deprecated, so it would be better to say "before assigning back to [member polygon]" instead of "before calling set_polygon".